state/state.go

```diff
@@ -103,26 +102,6 @@ type State struct {
 	CloudImageMetadataStorage cloudimagemetadata.Storage
 }
 
-// StateServingInfo holds information needed by a controller.
```

Comment:
I feel that we should have the document type that is stored in the database defined here in state. But expose to the world the controller.StateServingInfo

Reply:
yup